src/utils/GitHubUtils/GitHubUtils.js

```diff
@@ -147,6 +169,7 @@ export const getJsonFormattedCalendarAsync = async (
   return parse(userData.rawUserSvg.outerHTML).then(parsedGitHubCalendar => ({
     parsedCalendar: parsedGitHubCalendar,
     error: false,
-    errorMessage: null
+    errorMessage: null,
+    timeFrame: Array.isArray(gitHubUsername) ? gitHubUsername[1] : null
```

Comment:
One thing I'd like to point out here is that this logic should be handled by the caller imo(i.e. this function should behave as before including the returned object as well).